medusa/process_tv.py

```diff
@@ -453,26 +457,17 @@ def unrar(self, path, rar_files, force=False):
                             break
 
                     if not skip_extraction:
-                        rar_handle.extract(path=path, withSubpath=False, overwrite=False)
+                        rar_handle.extract(path)
```

Comment:
I'd like to point out two things here:
1) Although this works, according to the documentation .extractall() should be used to extract all files, while .extract() should be used to extract a single file.
2) This modification changes the current behavior. While with unrar2 subfolders were discarded (withSubpath=False), this is not the case anymore.